up-l1/README.adoc

```diff
@@ -294,6 +295,7 @@ UTransport implementations
 
 * *MUST* fail with a `UCode.UNIMPLEMENTED` if the transport does not support the _push_ <<delivery-method>>. In that case, the <<register-listener>> method *MUST* also fail accordingly.
 * *MUST* fail with a `UCode.NOT_FOUND`, if no such listener had been registered before
+* *MUST* be idempotent, multiple calls to said API with the same parameters *MUST* have the same effect as a single call.
```

Comment:
The following call would return NOT_FOUND if initial call success?

Reply:
OMG right (so dumb) fixed it now thanks @dddj698 !

Reply:
Removed the unregisterListener idempotent requirement